resources/views/layouts/home.blade.php

```diff
@@ -36,35 +38,39 @@
+                        <li class="{{ Route::currentRouteNamed('register') ? 'active' : '' }}" role="presentation">
+                            <a href="{{ route('register') }}">Register</a></li>
+                        <li class="{{ Route::currentRouteNamed('login') ? 'active' : '' }}" role="presentation">
+                            <a href="{{ route('login') }}">Login</a></li>
                     @else
                         @if (Auth::user()->isManager())
                             <li class="dropdown">
                                 <a class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false" href="{{ route('admin') }}">Manage <span class="caret"></span></a>
                                 <ul class="dropdown-menu" role="menu">
-                                    <li class="{{ Route::currentRouteNamed('user.index') ? 'active' : '' }}" role="presentation">
-                                        <a href="{{ route('user.index') }}"><i class="fa-fw fa fa-user" aria-hidden="true"></i> Manage Users</a></li>
                                     <li class="{{ Route::currentRouteNamed('device.index') ? 'active' : '' }}" role="presentation">
                                         <a href="{{ route('device.index') }}"><i class="fa-fw fa fa-microchip" aria-hidden="true"></i> Manage Devices</a></li>
 									<li class="{{ Route::currentRouteNamed('location.index') ? 'active' : '' }}" role="presentation">
 										<a href="{{ route('location.index') }}"><i class="fa-fw fa fa-map-marker" aria-hidden="true"></i> Manage Locations</a></li>
-									<li class="{{ Route::currentRouteNamed('site.index') ? 'active' : '' }}" role="presentation">
-										<a href="{{ route('site.index') }}"><i class="fa-fw fa fa-globe" aria-hidden="true"></i> Manage Sites</a></li>
-                                    <li class="{{ Route::currentRouteNamed('sensor.index') ? 'active' : '' }}" role="presentation">
-                                        <a href="{{ route('sensor.index') }}"><i class="fa-fw fa fa-line-chart" aria-hidden="true"></i> Manage Sensors</a></li>
                                     <li class="{{ Route::currentRouteNamed('logs') ? 'active' : '' }}" role="presentation">
                                         <a href="{{ route('logs') }}"><i class="fa-fw fa fa-history" aria-hidden="true"></i> View Logs</a></li>
+									<li class="{{ Route::currentRouteNamed('site.index') ? 'active' : '' }}" role="presentation">
+										<a href="{{ route('site.index') }}"><i class="fa-fw fa fa-globe" aria-hidden="true"></i> Manage Sites</a></li>
+									<li class="{{ Route::currentRouteNamed('user.index') ? 'active' : '' }}" role="presentation">
+                                        <a href="{{ route('user.index') }}"><i class="fa-fw fa fa-user" aria-hidden="true"></i> Manage Users</a></li>
                                 </ul>
```

Comment:
Should the manage dropdown just list the names without the manage in front of every item? Like instead of `Manage Devices` do `Devices`. Just a thought.

Reply:
fixed in 59cc6b1